### PR TITLE
Last five helper utils are linted

### PR DESCRIPTION
**Pull request by toni-leigh**
## Description

* All the components have one line changed for the loading of the tags helper, it's now a default import

# TODO
- [x] Release notes

## Testing Instructions

I would suggest Dev QA on this PR as it requires implementation

* Serialize Helper is called when a form is saved and submitted. This could be tested on the demo site but it doesn't currently save due to form validation - ✅ 
* Should Component Update decorator is not used in Carbon - ✅ 
* Tags is used for all components. General regression on a few should be fine. The tags that are generated are the ones used for automation - ✅ 
* toArray is again not used in Carbon  ✅

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -59,6 +59,15 @@ The following utils have had minor internal changes to satisfy the introduction
 * Router
 * Service
 
+The following utils have had minor internal changes to satisfy the introduction of stricter linting rules:
+
+* SerializeHelper
+* ShouldComponentUpdateHelper
+* TagsHelper
+* ToArrayHelper
+* ValidationsHelper
```
Comment (ianoxley) [inline CHANGELOG.md]: Do we need to list all the helpers, or can we just write 'Helpers'?
(see https://github.com/Sage/carbon/pull/1338/files?short_path=4ac32a7#diff-4ac32a78649ca5bdd8e0ba38b7006a1e - this matches how we've listed Validations without listing each individual validation).

File: src/utils/helpers/should-component-update/should-component-update.js [hunk 2]
```diff
@@ -35,7 +35,6 @@ function customCheck(current, next, key) {
   // allow isEqual to continue as normal
   if (Immutable.Iterable.isIterable(current)) {
     return Immutable.is(current, next);
-  } else {
-    return void 0;
   }
+  return undefined;
```
Comment (abramin) [inline src/utils/helpers/should-component-update/should-component-update.js]: Is this correct? shouldn't it return `false`?

Comment (toni-leigh) [inline src/utils/helpers/should-component-update/should-component-update.js]: This replaced `return void`, so not sure?

Comment (abramin) [inline src/utils/helpers/should-component-update/should-component-update.js]: I think return `void` was wrong in this case, `void ` should be used if a method is being called but nothing is being returned. 
`return null` indicates that no method is called and nothing is being returned either.

Comment (abramin) [inline src/utils/helpers/should-component-update/should-component-update.js]: Ok will leave as is

**Pull request merged**